composeApp/src/androidMain/kotlin/it/fast4x/rimusic/ui/screens/player/Lyrics.kt

```diff
@@ -351,15 +350,14 @@ fun Lyrics(
                             destLanguage,
                             Language.AUTO
                         )
```

Comment:
It is possible to have at most one translation call, but this requires a (kind of) complicated change in the "output section". Should I do that?